play-json/native/src/main/scala/StaticBinding.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2009-2021 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
I figure this is needed to ensure linting works on the new rules. But this isn't really associated with Lightbend anymore. @mkurz we should visit revamping the requirements on copyrights and file headers.

Reply:
I will do that in a upcoming pr.